### PR TITLE
Refactor/require to revert

### PR DESCRIPTION
**Pull request by LordCheta**
This PR aims to introduce more gas savings by using reverts instead of require statements

### CODE REVIEW

File: contracts/Registrar/PartnerRegistrar.sol [hunk 1]
```diff
@@ -267,7 +278,10 @@ contract PartnerRegistrar is
         if (partnerConfiguration.getMinCommitmentAge() == 0) {
             revert("Commitment not required");
         }
-        require(_commitmentRevealTime[commitment] < 1, "Existent commitment");
+
+        if (_commitmentRevealTime[commitment] > 1) {
```
Comment (OnyekaIjeh) [inline contracts/Registrar/PartnerRegistrar.sol:282]: This should be `> 0`

**Review by OnyekaIjeh — CHANGES_REQUESTED** [review]

File: contracts/Registrar/PartnerRegistrar.sol (hunk 1 above)
Comment (LordCheta) [reply contracts/Registrar/PartnerRegistrar.sol]: Thanks for catching that

**Review by sergio-jara — APPROVED** [review]
nice. love this PR. by removing the require statements we can actually tell if our tests are covering these edge cases.

**Review by OnyekaIjeh — APPROVED** [review]
